libs/tokens/src/hooks/tokens/unsupported/useUnsupportedTokens.ts

```diff
@@ -1,7 +1,8 @@
 import { useAtomValue } from 'jotai'
 
 import { currentUnsupportedTokensAtom } from '../../../state/tokens/unsupportedTokensAtom'
+import { UnsupportedTokensState } from '@cowprotocol/tokens'
 
-export function useUnsupportedTokens() {
-  return useAtomValue(currentUnsupportedTokensAtom)
+export function useUnsupportedTokens(): UnsupportedTokensState {
+  return useAtomValue(currentUnsupportedTokensAtom) || {}
```

Comment:
Given the recent issue returning a new array on every render, shouldn't this also have a stable empty value declared outside of the hook?